apps/user_status/src/UserStatus.vue

```diff
@@ -283,6 +283,7 @@ export default {
 			margin-right: 10px;
 			opacity: 1 !important;
 			background-size: 16px;
+			vertical-align: middle;
```

Comment:
```suggestion
			vertical-align: middle !important;
```

Otherwise it's overwritten by input.scss

![Bildschirmfoto vom 2022-10-17 16-38-02](https://user-images.githubusercontent.com/213943/196206248-d3c1692b-d3b1-4e9b-8f31-9ebad9775210.png)

Resulting in the same layout:

![Bildschirmfoto vom 2022-10-17 16-36-44](https://user-images.githubusercontent.com/213943/196206245-f7e97b48-99a7-423a-b7f0-410acd4b8d45.png)